degirum_tools/notifier.py

```diff
@@ -13,6 +13,8 @@
 from .ui_support import put_text, color_complement, deduce_text_color, CornerPosition
 from typing import Tuple, Union, Optional
 import time
+from apprise import Apprise

```

Comment:
Also we need to add the package to requirements.txt

Reply:
That would mean just calling pip freeze > requirements.txt upon adding a new library right? Given that my venv is properly set up for the project, of course.